packages/react/components/tabs/item/TabsItem.tsx

```
@@ -1,121 +1,51 @@
-        href={href}
-        className={classes}
+  return (
+    <>
+      <input
+        role='tab'
+        type="radio"
+        id={id}
+        name={groupName}
+        defaultChecked={active}
+        className={hashClass(styled, clsx('tab-input'))}
+        onClick={(e) => onClick && onClick(e)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)